src/assets/ba_data/python/babase/__init__.py

```diff
@@ -200,8 +201,10 @@
 from babase._workspace import WorkspaceSubsystem
 
 _babase.app = app = App()
+_babase.discord = discord = Discord()
```

Comment:
I'd rather that 'app' be the only object instance directly under babase, so can we store this as `babase.app.discord`? So you'd instantiate it in _app.py instead of here (alongside the various other subsystems).